Mathlib/GroupTheory/GroupAction/MultipleTransitivity.lean

```diff
@@ -284,6 +292,114 @@ end Higher
 
 end MulAction
 
+namespace Equiv.Perm
+
+variable {α : Type*} [Fintype α]
+
+-- Move?
+/-- The permutation group of `α` acts transitively on `α`. -/
+instance : IsPretransitive (Perm α) α where
+  exists_smul_eq x y := by
+    classical
+    use Equiv.swap x y
+    simp only [Equiv.Perm.smul_def, Equiv.swap_apply_left x y]
```

Comment:
Can you unsqueeze?

Reply:
what do you mean with "unsqueeze"?

Reply:
I tagged docs#MulAction.IsPretransitive with mk_iff.